src/mixins/GalleryHelpers.js

```diff
@@ -0,0 +1,111 @@
+import axios from 'axios'
```

Comment:
Can't see this file in use anywhere?

Reply:
No, this is intended to be a helper for third party libraries but not for this library itself. I will move the getRequest as part of the mixin for the card component tho.